src/app/components/client/toolbar/images/menu-icon-contact.svg

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   width="24"
+   height="24"
+   viewBox="0 0 24 24"
+   fill="none"
+   version="1.1"
+   id="svg1"
+   sodipodi:docname="menu-icon-contact.svg"
+   inkscape:version="1.3.2 (091e20ef0f, 2023-11-25)"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg">
+  <defs
+     id="defs1" />
+  <sodipodi:namedview
+     id="namedview1"
+     pagecolor="#ffffff"
+     bordercolor="#000000"
+     borderopacity="0.25"
+     inkscape:showpageshadow="2"
+     inkscape:pageopacity="0.0"
+     inkscape:pagecheckerboard="0"
+     inkscape:deskcolor="#d1d1d1"
+     inkscape:zoom="29.433333"
+     inkscape:cx="1.5798414"
+     inkscape:cy="11.908267"
+     inkscape:window-width="1920"
+     inkscape:window-height="1131"
+     inkscape:window-x="0"
+     inkscape:window-y="0"
+     inkscape:window-maximized="1"
+     inkscape:current-layer="svg1" />
+  <path
+     fill-rule="evenodd"
+     clip-rule="evenodd"
+     d="m 6.423,4.5000577 h 11.154 c 1.3375,0.00165 2.4213,1.0855 2.423,2.423 v 6.1540003 c -0.0017,1.3375 -1.0855,2.4213 -2.423,2.423 H 17 v 3 c -10e-5,0.4227 -0.2659,0.7996 -0.6639,0.9417 -0.3981,0.142 -0.8425,0.0184 -1.1101,-0.3087 l -2.966,-3.633 H 6.423 C 5.0855,15.498358 4.0016529,14.414558 4,13.077058 V 6.9230577 c 0.00165,-1.3375 1.0855,-2.4213471 2.423,-2.423 z M 17.8761,13.376168 C 17.9554,13.296838 18,13.189248 18,13.077058 V 6.9230577 c 0,-0.23362 -0.1894,-0.423 -0.423,-0.423 H 6.423 c -0.23362,0 -0.423,0.18938 -0.423,0.423 v 6.1540003 c 0,0.11219 0.04457,0.21978 0.12389,0.29911 0.07933,0.07932 0.18692,0.12389 0.29911,0.12389 h 6.311 c 0.30034,-2.5e-4 0.58489,0.13449 0.775,0.367 l 1.491,1.827 v -1.194 c 0,-0.55229 0.4477,-1 1,-1 h 1.577 c 0.1122,0 0.2198,-0.04457 0.2991,-0.12389 z M 15.5,8.5000577 h -7 c -0.27614,0 -0.5,0.22386 -0.5,0.5 0,0.27614 0.22386,0.5 0.5,0.5 h 7 c 0.2761,0 0.5,-0.22386 0.5,-0.5 0,-0.27614 -0.2239,-0.5 -0.5,-0.5 z m -7,2.0000003 h 7 c 0.2761,0 0.5,0.22386 0.5,0.5 0,0.27614 -0.2239,0.5 -0.5,0.5 h -7 c -0.27614,0 -0.5,-0.22386 -0.5,-0.5 0,-0.27614 0.22386,-0.5 0.5,-0.5 z"
+     fill="#3D3D3D"
+     id="path1"
+     style="fill:#592acb;fill-opacity:1" />
+</svg>
```

Comment:
This closing tag is highlighting red in my GitHub, but it passes SVG and XML validators, so might just be a _GitHub Thing&trade;_. 🤷

Reply:
Yeah not sure what's going on there, 3ff853417 doesn't fix it either. But yeah, if it validates and renders correctly, I'm going to chalk it up to GitHub.